app/helpers/blacklight/facets_helper_behavior.rb

```diff
@@ -8,7 +8,7 @@ module Blacklight::FacetsHelperBehavior
   # @param [Array<String>] fields
   # @param [Hash] options
   # @return [Boolean]
-  def has_facet_values? fields = facet_field_names, options = {}
```

Comment:
I don't know how downstream applications would use `options` here; maybe we can just drop it as part of this major release?